### PR TITLE
Allow version to be chosen

### PR DESCRIPTION
**Pull request by arvindsv**
- Use "latest" to always upgrade to latest.
- Otherwise, use a full version like: "2.46-15.el6".

### CODE REVIEW

**Comment by arvindsv** [comment]
I put this in a cookbooks directory and then downloaded the dependent "yum" cookbook and checked:

```
mkdir cookbooks
mv nsis-cookbook cookbooks
cd cookbooks
knife cookbook -z site download yum
tar xvf yum-*.tar*
cd ..
chef-client -z -o nsis-cookbook
```

Happy to be told there are easier ways to check this.

**Comment by ketan** [comment]
Update the README section about the node attribute?

**Comment by ketan** [comment]
Oh and changelog, and bump version in metadata so I can just push it to chef supermarket :)

**Comment by arvindsv** [comment]
Task master! :)

Should I make the version 0.1.1 or 0.2.0?

**Comment by arvindsv** [comment]
I've made it 0.2.0. Let me know if you want it to be 0.1.1 instead.

File: metadata.rb [hunk 1]
```diff
@@ -1,10 +1,10 @@
 name             'nsis'
 maintainer       'Ketan Padegaonkar'
 maintainer_email 'ketanpadegaonkar@gmail.com'
-license          'MIT'
+license          'Apache v2.0'
```
Comment (ketan) [inline metadata.rb:4]: That is supposed to be `Apache-2.0` which is the OSI approved code :) The rest looks ok and I'll merge it.

Comment (arvindsv) [inline metadata.rb:4]: I got that value from the docs here: https://docs.chef.io/cookbook_repo.html

It said "Apache v2.0". :)

**Pull request merged**